app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +72,137 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("The board can bring a cell to life at a given location")
+    void setAliveAt_When1_1IsPassed_ReturnsTheCellObjectAt1_1() {
+        Board board = new Board();
+
+        Boolean cellStatus = board.isCellAliveAt(1, 1);
+        assertFalse(cellStatus);
+
+        board.setAliveAt(1, 1);
+        cellStatus = board.isCellAliveAt(1, 1);
+
+        assertTrue(cellStatus);
+
+    }
+
+    @Nested
+    @DisplayName("getNumberOfAliveNeighbors()")
+    class GetNumberOfAliveNeighbors {
+        private void setDeadBoard() {
+            for ( ArrayList<Cell> gridRow : board.getGrid()) {
+                for ( Cell cell : gridRow) {
+                    cell.die();
+                }
+            }
+        }
+
+        private void setAliveBoard() {
+            for ( ArrayList<Cell> gridRow : board.getGrid()) {
+                for (Cell cell : gridRow) {
+                    cell.revive();
+                }
+            }
+        }
+
+        public Board board;
+        @BeforeEach
+        void createBoard() {
+            board = new Board(3,3);
+            setDeadBoard();
+        }
+        @Test
+        @DisplayName("A cell with 0 alive neighbor should return 0 alive neighbor")
+        void getNumberOfAliveNeighbors_WhenNoNeighborsAreAlive_Returns0() {
+            int numOfAliveNeighbors = board.getNumberOfAliveNeighbors(1,1);
+            assertEquals(numOfAliveNeighbors, 0);
+        }
+
+        @Test
+        @DisplayName("A cell with 1 alive neighbor should return 1 alive neighbor")
+        void getNumberOfAliveNeighbors_When1NeighborIsAlive_Returns1() {
+            board.setAliveAt(0,1);
```

Comment:
It's common practice to use a space in between multiple arguments after the comma. e.g. `board.setAlive(0, 1)`

I notice it's being done at some places, but not consistently everywhere like in this example.

Reply:
When I add arguments or pass parameters I literally say 'remember Jeff strongly dislikes missing spaces" Im surprised I let a few past. But I went back in and made the proper fixes